### PR TITLE
Create FOXE-FOXEERH743.config

### PR DESCRIPTION
**Pull request by Johnsnow-foxeer**
Would like to add Foxeer H743 target based on MPU6000, thank you

### CODE REVIEW

**Comment by sugaarK** [comment]
Hey @Johnsnow-foxeer  please read the unified target doc as the process has changed alittle and you need to add defines for our cloud builder which are missing from your .config https://github.com/betaflight/betaflight/blob/master/docs/TargetMaintenance/CreatingAUnifiedTarget.md

**Review by Johnsnow-foxeer — COMMENTED** [review]
Adding the defines for the flash chip, planned gyros Baros etc

File: configs/default/FOXE-FOXEERH743.config [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+# pin D13: DMA1 Stream 7 Request 30
+dma pin C08 2
+# pin C08: DMA1 Stream 2 Request 49
+dma pin C09 3
+# pin C09: DMA1 Stream 3 Request 50
+
+# feature
+feature RX_SERIAL
+feature TELEMETRY
+feature LED_STRIP
```
Comment (sugaarK) [inline configs/default/FOXE-FOXEERH743.config:127]: this is now a user define so remove as it can be not included in the build

**Review by sugaarK — CHANGES_REQUESTED** [review]

File: configs/default/FOXE-FOXEERH743.config (hunk 1 above)
Comment (sugaarK) [reply configs/default/FOXE-FOXEERH743.config:127]: LED_STRIP

**Review by Johnsnow-foxeer — COMMENTED** [review]
Remove line feature LED_STRIP

File: configs/default/FOXE-FOXEERH743.config (hunk 1 above)
Comment (Johnsnow-foxeer) [reply configs/default/FOXE-FOXEERH743.config]: got it

Comment (sugaarK) [reply configs/default/FOXE-FOXEERH743.config]: nice

**Review by sugaarK — APPROVED** [review]

**Review by KarateBrot — APPROVED** [review]

**Pull request merged**